uaclient/files/state_files.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Dict
+
+from uaclient.data_types import BoolDataValue, DataObject, Field
+from uaclient.files.data_types import DataObjectFile
+from uaclient.files.files import UAFile
+
+SERVICES_ONCE_ENABLED = "services-once-enabled"
+
+
+class _ServicesOnceEnabledData(DataObject):
+    fields = [
+        Field("fips_updates", BoolDataValue, False),
+    ]
+
+    def __init__(self, fips_updates: bool):
+        self.fips_updates = fips_updates
+
+
+class _ServiceOnceEnableDataObjectFile(DataObjectFile):
```

Comment:
I think this still needs to specify the TypeVar for DataObjectFile
```suggestion
class _ServiceOnceEnableDataObjectFile(DataObjectFile[_ServicesOnceEnabledData]):
```